transmart-rest-api/grails-app/controllers/org/transmartproject/rest/PatientSetController.groovy

```diff
@@ -1,16 +1,18 @@
 package org.transmartproject.rest
 
-import grails.web.mime.MimeType
+import grails.rest.Link
+import grails.rest.render.util.AbstractLinkingRenderer
+import org.codehaus.groovy.grails.web.mime.MimeType
```

Comment:
This does not look right, should definitely be `grails.web.mime.MimeType` in Grails 3.